src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,131 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import classNames from 'classnames';
+import { GenderFilters, SearchQueries } from '../../types';
+
+const centutriesToFilter = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const toggleCentruies = (centr: string): string => {
+    const curCentr = searchParams.getAll(SearchQueries.centuries);
+    const newCentr = curCentr.includes(centr)
+      ? curCentr.filter(c => c !== centr)
+      : [...curCentr, centr];
+
+    return getSearchWith(searchParams, { [SearchQueries.centuries]: newCentr });
+  };
```

Comment:
The function `toggleCenturies` is a bit confusing. It seems like you're toggling the inclusion of a century in a list of centuries. It would be clearer if you split this into two separate functions: `addCentury` and `removeCentury`. This would make the code easier to read and understand.